oauth2client/contrib/django_util/__init__.py

```diff
@@ -210,6 +264,20 @@ def _get_oauth2_client_id_and_secret(settings_instance):
                 "GOOGLE_OAUTH2_CLIENT_SECRET in settings.py")
 
 
+def _get_storage_model():
+    """Initializes setting to control whether to use a Django model to store
```

Comment:
I'm having a hard time understanding this particular docstring's wording. Do you think you could re-word it?